.github/CODEOWNERS

```diff
@@ -13,23 +13,16 @@
 /client/my-sites/comment* @Automattic/serenity
 
 # Email services
-/client/components/data/query-gsuite-users @Automattic/yolo
-/client/components/domains/contact-details-form-fields/index.jsx @Automattic/yolo
 /client/components/gsuite @Automattic/yolo
 /client/components/marketing-survey/gsuite-cancel-purchase-dialog @Automattic/yolo
 /client/lib/domains/email-forwarding @Automattic/yolo
 /client/lib/gsuite @Automattic/yolo
 /client/lib/titan @Automattic/yolo
-/client/me/purchases/remove-purchase/index.jsx @Automattic/yolo
-/client/my-sites/checkout/checkout-thank-you/index.jsx @Automattic/yolo
-/client/my-sites/checkout/checkout/domain-details-form.jsx @Automattic/yolo
-/client/my-sites/checkout/checkout/index.jsx @Automattic/yolo
+/client/my-sites/checkout/checkout-thank-you/index.tsx @Automattic/yolo
```

Comment:
```suggestion
```
Let's remove this. This line is super old, and Letero was never an official steward of this page, despite our work on them last year (pb5gDS-36j-p2).